app/src/main/res/layout/quicksettings_website_info.xml

```diff
@@ -35,24 +35,25 @@
             style="@style/QuickSettingsText"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:paddingStart="8dp"
             android:paddingTop="8dp"
-            tools:text="https://wikipedia.org" />
+            tools:text="https://wikipedia.org"
+            tools:ignore="RtlSymmetry" />
     </LinearLayout>
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:minHeight="@dimen/tracking_protection_item_height"
         android:orientation="horizontal"
-        style="@style/QuickSettingsText"
-        android:paddingTop="8dp"
-        android:paddingBottom="8dp">
+        style="@style/QuickSettingsText">
         <ImageView
             android:id="@+id/securityInfoIcon"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:importantForAccessibility="no"
             app:srcCompat="@drawable/mozac_ic_lock"
+            android:paddingStart="0dp"
             android:paddingEnd="8dp"/>
```

Comment:
It seems weird to me that we are adding the padding to the icon instead of the LinearLayout below to align the text. Although this is probably a more likely candidate to refactor into a ConstraintLayout.

Reply:
I just adding it to remove the lint warning, it already had the  `paddingEnd`.

![image](https://user-images.githubusercontent.com/773158/125345973-2fcb1480-e327-11eb-88ad-2307978bf7f8.png)